src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -339,7 +340,9 @@ private void triggerChangeExportMediaOverride() {
   }
 
   private boolean confirmOverwriteFiles() {
-    if (showConfirmDialog(this, "The default behavior is to append to existing files. Are you sure you want to overwrite existing files?", "", YES_NO_OPTION, PLAIN_MESSAGE) == YES_OPTION)
+    JDialog dialog = new JDialog();
+    dialog.setAlwaysOnTop(true);
+    if (showConfirmDialog(dialog, "The default behavior is to append to existing files. Are you sure you want to overwrite existing files?", "", YES_NO_OPTION, PLAIN_MESSAGE) == YES_OPTION)
```

Comment:
The parent of the overwrite dialog should be the Export Configuration Dialog, not some new invisible one with magic keep-on-top powers. 😀

Reply:
Hehe, good thing we were like “let’s wait until Dave looks at this before proceeding!” It always seems so obvious after you correct it...

Reply:
It was a pretty good kludge! 😀

Reply:
Sure, I’ll go along with that.

Reply:
Thanks, @dcbriccetti!